src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionServiceV1.java

```diff
@@ -577,16 +578,17 @@ public boolean isStageCompatible(final String stageName) {
     checkConnection();
     InternalUtils.assertNotEmpty("stageName", stageName);
     if (!stageExist(stageName)) {
-      LOGGER.debug("stage {} doesn't exists", stageName);
+      LOGGER.info("stage {} doesn't exists", stageName);
```

Comment:
I'm thinking how will you find this and above log? Imagine you want to search for it in a file. When you search by "stage" or "doesn't exists" you will find many of words like this. 
Maybe we should use some convention, adding some kind of a signature for easier search? Is my concern valid, WDYT?

Reply:
I would keep it on debug, similar to table one, and use the next one for INFO.